RusTaxi/Classes/Screens/Custom Views/Accept View/AcceptView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  AcceptView.swift
+//  RusTaxi
+//
+//  Created by Ruslan Prozhivin on 19.09.2018.
+//  Copyright © 2018 App's ID. All rights reserved.
+//
+
+import UIKit
+
+class AcceptView: UIView {
+	@IBOutlet weak var driverImageView: UIImageView!
+	@IBOutlet weak var nameLabel: UILabel!
+	@IBOutlet weak var carLabel: UILabel!
+	@IBOutlet weak var acceptButton: UIButton!
+	@IBOutlet weak var refuseView: UIView!
+	@IBOutlet weak var refuseButton: UIButton!
+	@IBOutlet weak var starImageView: UIImageView!
+	
+	override func awakeFromNib() {
+		super.awakeFromNib()
+		
+		customizeImageView()
+		customizeButton()
+		customizeRefuseView()
+		customizeAcceptView()
+		hideStarImage()
+	}
+	
+	private func customizeAcceptView() {
+	}
+	
+	private func customizeImageView() {
+		driverImageView.layer.masksToBounds = false
+		driverImageView.layer.cornerRadius = driverImageView.frame.height/2
```

Comment:
.cornerRadius related by frame should set inside `viewDidLayoutSubviews`
and space BETWEEN / and operands